device-types/HPE/HP-6200-24G-mGBIC-yl.yml

```diff
@@ -0,0 +1,66 @@
+---
+manufacturer: HPE
+model: HP 6200-24G-mGBIC yl
+slug: hpe-hp-6200-24g-mgbic-yl
+part_number: J8992A
+u_height: 1
+weight: 6.4
+weight_unit: kg
+comments: '[HPE 6200 yl Switch Series](https://support.hpe.com/hpesc/public/docDisplay?docId=c01803645&docLocale=en_US)'
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: de-9
+power-ports:
+  - name: ps1
+    type: iec-60320-c14
+    maximum_draw: 243
+interfaces:
+  - name: '1'
+    type: 1000base-t
```

Comment:
If this is a GBIC model are these supposed to be SFP ports? If so they are `type: 1000base-x-sfp`